README.md

```diff
@@ -25,12 +25,23 @@ Abi-wan-kanabi is an UNLICENSE standalone TypeScript parser for Cairo smart cont
 It enables on the fly typechecking and autocompletion for contract calls directly in TypeScript.
 Developers can now catch typing mistakes early, prior to executing the call on-chain, and thus enhancing the overall Dapp development experience.
 
+### Cairo versions
+
+Abiwan will support multiple Cairo compiler versions, but not in parallel - different package versions will support consecutive Cairo versions.
+
+Currently supported:
+
+| Abi-Wan npm                                                   | Cairo compiler                                                                                                                                               |
+| ------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [1.0.3](https://www.npmjs.com/package/abi-wan-kanabi/v/1.0.3) | [Cairo v1.0.0](https://github.com/starkware-libs/cairo/releases/tag/v1.0.0) <br> [Cairo v1.1.0](https://github.com/starkware-libs/cairo/releases/tag/v1.1.0) |
+| [2.1.0](https://www.npmjs.com/package/abi-wan-kanabi/v/2.1.0) | [Cairo v2.3.0](https://github.com/starkware-libs/cairo/releases/tag/v2.3.0)                                                                                  |
```

Comment:
what about [v2.4.0](https://github.com/starkware-libs/cairo/releases/tag/v2.4.0) ? I think it should work as well, no ABI changes? If so we can note it here!

Reply:
Let me check if it works, if so, I'll update it

Reply:
Even though there is no ABI change, they introduced new types https://github.com/starknet-io/starknet.js/pull/891